python_modules/dagster/dagster/_core/definitions/asset_reconciliation_sensor.py

```diff
@@ -0,0 +1,332 @@
+
+    * If a, b, and c are all materialized, then on the next sensor tick, the sensor will see that d and e can
+        be materialized. Since d and e will be materialized, f can also be materialized. The sensor will kick off a
+        run that will materialize d, e, and f.
+    * If on the next sensor tick, a, b, and c have not been materialized again the sensor will not launch a run.
+    * If before the next sensor tick, just asset a and b have been materialized, the sensor will launch a run to
+        materialize d.
+    * If asset c is materialized by the next sensor tick, the sensor will see that e can be materialized (since b and
+        c have both been materialized since the last materialization of e). The sensor will also see that f can be materialized
+        since d was updated in the previous sensor tick and e will be materialized by the sensor. The sensor will launch a run
+        the materialize e and f.
+    * If by the next sensor tick, only asset b has been materialized. The sensor will not launch a run since d and e both have
+        a parent that has not been updated.
+    * If during the next sensor tick, there is a materialization of a in progress, the sensor will not launch a run to
+        materialize d. Once a has completed materialization, the next sensor tick will launch a run to materialize d.
+
+    Other considerations:
+    * If an asset has a SourceAsset as a parent, the sensor will always consider the source asset to updated
+        since the previous sensor tick. If you have the asset graph my_source->my_asset where my_source is
+        a SourceAsset. The sensor created from build_asset_reconciliation_sensor(AssetSelection.assets(my_asset), name="my_sensor")
+        will materialize my_asset on every sensor tick.
+
+    Args:
+        selection (AssetSelection): The group of assets you want to monitor
+        name (str): The name to give the sensor.
+        and_condition (bool): If True (the default) the sensor will only materialize an asset when
+            all of its parents have materialized. If False, the sensor will materialize an asset when
+            any of its parents have materialized.
+        parent_in_progress_stops_materialization (bool): If True, the sensor will not materialize an
+            asset there is an in-progress run that will materialize any of the asset's parents. Defaults to
+            False.
+        minimum_interval_seconds (Optional[int]): The minimum amount of time that should elapse between sensor invocations.
+        description (Optional[str]): A description for the sensor.
+        default_status (DefaultSensorStatus): Whether the sensor starts as running or not. The default
+            status can be overridden from Dagit or via the GraphQL API.
+
+    Returns: A MultiAssetSensor that will monitor the parents of the provided assets to determine when
```

Comment:
MultiAssetSensor isn't the name of a class, right?

Reply:
it's not - my bad. Should be MultiAssetSensorDefinition